.docker/plugins/s3fs/config.json

```diff
@@ -21,6 +21,30 @@
         },
         {
           "Description": "",
+          "Name": "LIBSTORAGE_INTEGRATION_VOLUME_OPERATIONS_MOUNT_ROOTPATH",
+          "Settable": [
+            "value"
+          ],
+          "Value": "/data",
```

Comment:
The comma here is not necessary, as it's the last item in the map.  Same with the other two.